src/settings/SettingsSelfExclusion.js

```diff
@@ -88,6 +91,7 @@ export default class SettingsSelfExclusion extends PureComponent {
         success: true,
         serverError: false,
       });
+      document.getElementsByClassName('update-notice')[0].scrollIntoView();
```

Comment:
@arnabk @aminroosta guys. I have here a preview of using refs instead of getElementById: https://github.com/brucebinary/binary-next-gen/compare/scroll-to-error-fix...brucebinary:scroll-to-error-finddomnode My point is to demonstrate that using refs in this context is tedious and difficult to understand, for something that is seemingly trivial. What is happening is that the we are still accessing the DOM node, but instead of doing it directly (react components don't have a `scrollIntoView` method, unless you install a library for it) via `document` we access it through the corresponding react component via `findDOMNode` (use of `findDOMNode` is actually ill-advised and eslint will throw errors for it). 

However, if everyone agrees to do it this way I will update this PR accordingly.

However I'm a little stuck with the above line of code. What it does it scroll to the update notice when the settings have successfully updated. Should we decide to use refs, the ref to this `UpdateNotice` component will need to be passed to this component, in which we again use it to find the corresponding DOM node via `findDOMNode` and call `scrollIntoView`.

Bear in mind all we want to do is to scroll the view to a specified input field.